CSharp/BatchExplorer/Helpers/Common.cs

```diff
@@ -40,14 +40,26 @@ public static class Common
                 "A7",
                 "A8",
                 "A9",
+                "A10",
+                "A11",
                 "STANDARD_D1",
+                "STANDARD_D1_V2",
                 "STANDARD_D2",
+                "STANDARD_D2_V2",
                 "STANDARD_D3",
-                "STANDARD_D4",
```

Comment:
It looks like you might have inadvertently changed STANDARD_D4 to STANDARD_D5, which according to https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-windows-sizes#d-series doesn't exist?